jobs/config.json

```diff
@@ -5415,19 +5425,21 @@
   },
   "ci-kubernetes-e2e-gke-cvm-1-5-cvm-1-6-upgrade-cluster": {
     "args": [
+      "--check-leaked-resources",
+      "--check-version-skew=false",
+      "--cluster=",
+      "--deployment=bash",
```

Comment:
As mentioned in the PR description, there's a couple here that need more work. This is a note of such (that I've looked at it and it can't move just yet).